apps/condo/migrations/20211229103947-0091_auto_20211229_0539.js

```diff
@@ -0,0 +1,55 @@
+// auto generated by kmigrator
```

Comment:
Remove your two migrations and create one! Rebesit from the master!

Reply:
fixed